notebooks/1-sequence-stats.jmd

```diff
@@ -0,0 +1,147 @@
+using Plots: make_fillrange_from_ribbon
```

Comment:
Assume this is meant to be in a code block?

Reply:
I didn't write this. I think it got added automatically somehow. Should I move it inside the code block?

Reply:
Hmm, that's odd - maybe something added by the julia extension? Doesn't look like you actually use the function it's importing. Just delete it I think